src/libusermode/running.cpp

```diff
@@ -492,8 +494,11 @@ void drakvuf_request_userhook_on_running_process(
     callback_t cb,
     void* extra)
 {
-    if (!instance || !instance->initialized)
-        throw -1;
+    // Userhook is a singleton.
+    if (!instance)
+        instance = new userhook(drakvuf);
+    if (!instance->initialized)
+        instance->init(drakvuf);
```

Comment:
This is not how singletons should be implemented, but that's not the cope of this PR.